File: Assignment_11/question2.c
///////////////////////////////////////////////////////////////////////////////////
//
//  File Name   : question2.c
//  Description : Display number and # in pattern.
//
//  Problem     : Accept number from user and display pattern: 5 # 4 # 3 # 2 # 1 #
//
//  Author      : Asawari Jadhav
//  Date        : 20/05/2025
//
//////////////////////////////////////////////////////////////////////////////////

#include <stdio.h>

void Pattern(int iNo)
{
  int iCnt = 0;
  for (iCnt = iNo; iCnt >= 1; iCnt--)
  {
    printf("%d\t#\t", iCnt);
  }
}

int main()
{
  int iValue = 0;

  printf("Enter number of elements\n");
  scanf("%d", &iValue);

  Pattern(iValue);

  return 0;
}

File: Assignment_47/question2.java
//////////////////////////////////////////////////////////////////////////////////////////////
//
//  File Name     : question2.java
//  Description   : Accept number of rows and columns from user and display pattern
//  Problem       : Display numbers 1 to iCol in each row
//
//  Author        : Asawari Jadhav
//
//  Date          : 19-08-2025            
//
//////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Pattern
{
    public void DisplayPattern(int iRow, int iCol)
    {
        for (int i = 0; i < iRow; i++)
        {
            for (int j = 1; j <= iCol; j++)
            {
                System.out.print(j + " ");
            }
            System.out.println();
        }
    }
}

public class question2
{
    public static void main(String args[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter number of rows : ");
        int iRow = sobj.nextInt();

        System.out.print("Enter number of columns : ");
        int iCol = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.DisplayPattern(iRow, iCol);

        
    }
}
